### DISCUSSION TITLE
上来夸一下win7可以用了，但是

### DISCUSSION THREAD

**Discussion by woodchuckk** [getlantern/lantern-forum-cn#251] [Q&A]
但是麻烦把各地服务器节点再稳定一下，偶尔还是会有频繁跳服务器节点链接不上的情况，请多关注一下，谢谢。

**Comment by ycjdixon** [comment]
最近确实非常不稳定，还没有免费节点稳定。频繁连不上、断线。

**Comment by MaxMKMax** [comment]
@woodchuckk @ycjdixon 
谢谢您上来夸奖我们一下!  嗯嗯, 同意你们的意见. 我们完全明白这个痛点。 我也无数次反馈给团队了。他们都听进去了! 
我们实际上计划创建功能供用户选择稳定的服务器。请耐心等待，我相信很快就能完成。
